examples/main.tf

```diff
@@ -0,0 +1,9 @@
+module "ingress_nginx" {
+  source                   = "../"
+  chart_version            = "4.8.3"
+  namespace                = "ingress-nginx"
+  host                     = "https://test"
+  cluster_ca_certificate   = base64decode("certificate")
+  token                    = "token"
+  lifecycle_ignore_changes = ["test"]
+}
```

Comment:
Please add an empty line here. To prevent this kind of issue please activate the auto-formatter.